ironfish/src/rpc/routes/chain/followChain.ts

```diff
@@ -117,7 +119,7 @@ router.register<typeof FollowChainStreamRequestSchema, FollowChainStreamResponse
         return transaction.withReference(() => {
           return {
             hash: BlockHashSerdeInstance.serialize(transaction.unsignedHash()),
-            size: Buffer.from(JSON.stringify(transaction.serialize())).byteLength,
+            size: bufio.sizeVarBytes(transaction.serialize()),
```

Comment:
shouldn't this be just the byte length of this, or else it'll have 2 extra bytes?